### PR TITLE
Added labels to the map

### PR DESCRIPTION
**Pull request by philippewarren**
First pass for #28.

### CODE REVIEW

File: opentera_webrtc_ros/scripts/goal_manager.py [hunk 1]
```diff
@@ -1,134 +1,61 @@
...
-
-    def clearGlobalPathClient(self):
-        rospy.wait_for_service('clear_global_path')
-        try:
-            clear_global_path = rospy.ServiceProxy(
-                'clear_global_path', SetBool)
-            clear_global_path(True)
-        except rospy.ServiceException as e:
-            rospy.logwarn("Service call failed: %s" % e)
-
```
Comment (philippewarren) [inline opentera_webrtc_ros/scripts/goal_manager.py:131]: A bunch of stuff was moved library files, to be shared with labels_manager.py

File: opentera_webrtc_ros/scripts/libmapimageconverter.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    pass
+    # rospy.wait_for_service('map_goal_to_image_goal')
+    # try:
+    #     map_goal_to_image_goal = rospy.ServiceProxy(
+    #         'map_goal_to_image_goal', MapGoalToImageGoal)
+    #     res: MapGoalToImageGoalResponse = map_goal_to_image_goal(
+    #         map_goal)
+    #     return res.map_goal
+    # except rospy.ServiceException as e:
+    #     rospy.logwarn("Service call failed: %s" % e)
```
Comment (philippewarren) [inline opentera_webrtc_ros/scripts/libmapimageconverter.py:49]: This service don't exist but is not needed currently

File: opentera_webrtc_ros/setup.py [hunk 3]
```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
+from setuptools import setup
```
Comment (philippewarren) [inline opentera_webrtc_ros/setup.py:3]: Modern way to do the same thing

File: opentera_webrtc_ros/scripts/libyamldatabase.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    def clear(self) -> None:
+        self.path.unlink(missing_ok=True)
+        self.path.touch()
+        self.refresh()
+
+    def values(self) -> Generator[T, None, None]:
+        return (v for v in self.data.values())
+
+    def _load(self) -> Dict[str, T]:
+        return self.yaml.load(self.path) or {}
```
Comment (philippewarren) [inline opentera_webrtc_ros/scripts/libyamldatabase.py:87]: YAML file database, could be reused if needed as it is pretty much generic, as long as T is dumpable/loadable

File: opentera_webrtc_ros/scripts/labels_manager.py [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+        pose.pose.orientation.x = float(
+            self.data["pose"]["pose"]["orientation"]["x"])
+        pose.pose.orientation.y = float(
+            self.data["pose"]["pose"]["orientation"]["y"])
+        pose.pose.orientation.z = float(
+            self.data["pose"]["pose"]["orientation"]["z"])
+        pose.pose.orientation.w = float(
+            self.data["pose"]["pose"]["orientation"]["w"])
+
+        return Label(name=self.data["name"], description=self.data["description"], pose=pose)
```
Comment (philippewarren) [inline opentera_webrtc_ros/scripts/labels_manager.py:50]: YAML dumpable/loadable representation of a Label

**Review by doumdi — APPROVED** [review]

**Review by doumdi — APPROVED** [review]

**Pull request merged**